tests/client/core/test_utils.js

```diff
@@ -516,7 +512,7 @@ describe('loadCategoriesIfNeeded', () => {
     return loadCategoriesIfNeeded(props).then(() => {
       assert(dispatch.calledWith(action), 'dispatch not called');
       mockApi.verify();
-      mockActions.verify();
+      return mockActions.verify();
```

Comment:
Yeah we could `return true` instead but a return is always required for any Promise.